app/controllers/sidekiq_monitoring/application_controller.rb

```diff
@@ -0,0 +1,5 @@
+module SidekiqMonitoring
+  class ApplicationController < SidekiqMonitoring.parent_controller.constantize
```

Comment:
neat

Reply:
I know, right? (stole it from `devise` 😇)